benefits/templates/admin/agency-base.html

```diff
@@ -26,6 +26,31 @@ <h1 class="p-0 m-0 text-white">Administrator</h1>
 {% endblock branding %}
 
 {% block usertools %}
+    {% if has_permission %}
+        <div id="user-tools">
+            {% block welcome-msg %}
+                <span class="text-uppercase text-white">
+                    {% translate "Welcome," %}
+                    <strong>{% firstof user.get_short_name user.get_username %}</strong>. </span>
```

Comment:
`<span class="fw-bold">` instead of `<strong>` - the <strong> doesn't make the font bold for me.

Reply:
Thanks, made the change in aa73dca7d3981c5bd06504cb9aa5d23011dfc39e